docs/getting_started/prerequisites.md

```diff
@@ -17,9 +17,9 @@ description: Prerequisites
 
 > For PrimeHub installation on **multiple nodes**, please read the prerequisites thoroughly. Before installing PrimeHub, we need to provide a PrimeHub-ready-Kubernetes, which should contain following essentials. Make sure the cluster is ready for PrimeHub.
 
-## Kubernetes Cluster (1.16+)
```

Comment:
I don't think 1.21 is our "prerequisites", we still support 1.16+. But the remaining change from 1.16 -> 1.21 is ok.

Side note:
And because 1.22 has some breaking changes, PrimeHub does not supoprt 1.22
https://kubernetes.io/blog/2021/07/14/upcoming-changes-in-kubernetes-1-22/

Reply:
For current minimum requirements for nginx, it is `v1.19`

https://github.com/kubernetes/ingress-nginx/blob/main/charts/ingress-nginx/Chart.yaml#L19

They made changes two months ago
https://github.com/kubernetes/ingress-nginx/blame/041eb167c7bfccb1d1653f194924b0c5fd885e10/charts/ingress-nginx/Chart.yaml

Reply:
alright, in terms of k8s version, will use 1.19+ instead.

Reply:
Others LGTM. You can merge once the change is made.